trinity/utils/chains.py

```diff
@@ -52,14 +56,8 @@ def get_data_dir_for_network_id(network_id: int) -> str:
         raise KeyError("Unknown network id: `{0}`".format(network_id))
 
 
-DATABASE_DIR_NAME = 'chain'
-
-
-def get_database_dir(data_dir: str) -> str:
-    """
-    Returns the directory path where chain data will be stored.
-    """
-    return os.path.join(data_dir, DATABASE_DIR_NAME)
+FULL_DATABASE_DIR_NAME = 'chain'
```

Comment:
I'd prefix this one too as `fullchain` so there is no ambiguity.  Or maybe even add an extra directory level and do `./chain/full/` and `./chain/light`